lntest/itest/lnd_signer_test.go

```diff
@@ -411,3 +412,75 @@ func deriveCustomizedKey(ctx context.Context, node *lntest.HarnessNode,
 	}
 	return pub, nil
 }
+
+// testSignVerifyMessage makes sure that the SignMessage RPC can be used with
+// all custom ways of specifying by verifying with VerifyMessage. Tests both
+// ecd and schnorr signatures.
+func testSignVerifyMessage(net *lntest.NetworkHarness, t *harnessTest) {
+	runSignVerifyMessage(t, net.Alice)
```

Comment:
Since you went with the `testXXX`/`runXXX` pattern, should we make sure this also works in the remote signing context and add this test to the `testRemoteSigner` test?

Reply:
That would make sense. I've added it to the watch-only wallet tests.